go.mod

```diff
@@ -3,10 +3,12 @@ module github.com/LUSHDigital/core
 go 1.14
 
 require (
+	github.com/LUSHDigital/core-lush v0.3.1
```

Comment:
Same for this, where are we using core-lush?

Reply:
pushed new patch to remove this